src/casync-tool.c

```diff
@@ -1058,25 +1058,28 @@ static int verbose_print_done_extract(CaSync *s) {
 
         if (n_local_requests != UINT64_MAX)
                 log_info("Chunk requests fulfilled from local store: %" PRIu64 " (%" PRIu64 "%%)",
-                         n_local_requests, n_local_requests * 100U / total_requests);
+                         n_local_requests,
+                         total_requests > 0 ? n_local_requests * 100U / total_requests : 0);
         if (n_local_bytes != UINT64_MAX)
                 log_info("Bytes used from local store: %s (%" PRIu64 "%%)",
                          format_bytes(buffer, sizeof(buffer), n_local_bytes),
-                         n_local_bytes * 100U / total_bytes);
+                         total_bytes > 0 ? n_local_bytes * 100U / total_bytes : 0);
```

Comment:
I noticed that there's such a division in `verbose_print_done_make`, the function just above.

Also, what about defining a macro?

```
#define PERCENT(val, total) (total > 0 ? value * 100U / total : 0)
```

I thought about it when I wrote this code, but in the end I didn't do it.

Reply:
Indeed. I'll submit a PR shortly.